lib/BusinessLogicService.js

```diff
@@ -20,8 +20,9 @@ const { CollectionHook, EndpointFieldNames, HTTPMethod, OperationType } = requir
 const { Endpoints, getObjectByOmitting, isNullOrUndefined, readFile } = require('./Utils');
 
 class BusinessLogicService extends BaseService {
-  getCommonCode(envId, name, done) {
-    const endpoint = Endpoints.commonCode(this.cliManager.config.defaultSchemaVersion, envId, name);
+  getCommonCode(envId, name, fields, done) {
+    let endpoint = Endpoints.commonCode(this.cliManager.config.defaultSchemaVersion, envId, name);
+    if (fields) endpoint += '?fields=' + encodeURIComponent(fields);
```

Comment:
You can set a `query` property on the req obj with value `fields=[fields]` or null and it will be handled before the request is sent